clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/description/servergroup/DeployDcosServerGroupDescription.groovy

```diff
@@ -47,6 +52,7 @@ class DeployDcosServerGroupDescription extends AbstractDcosCredentialsDescriptio
     String type
   }
 
+  // TODO don't think we need a lot of this stuff (fromContext, fromTrigger, cluster, account)
```

Comment:
There was a lot of stuff existing on the image description that are never used: stageId, cluster, account, fromContext, fromTrigger. Is there any reason we still need them here?